src/Auth0Client.ts

```diff
@@ -486,8 +486,11 @@ export default class Auth0Client {
    * auth0.logout();
    * ```
    *
-   * Performs a redirect to `/v2/logout` using the parameters provided
-   * as arguments. [Read more about how Logout works at Auth0](https://auth0.com/docs/logout).
+   * Clears the application session and performs a redirect to `/v2/logout`, using
+   * the parameters provided as arguments, to clear the Auth0 session.
+   * If the `federated` option is specified it also clears the Identity Provider session.
+   * If the `localOnly` option is specified, only clears the application session.
+   * [Read more about how Logout works at Auth0](https://auth0.com/docs/logout).
```

Comment:
Let's mention up-front here in the docs that specifying both options together is invalid and an error will be thrown, to complement the docs on the properties themselves.